common/k8stest/util_pvc.go

```diff
@@ -1331,3 +1331,184 @@ func GetPvCapacity(pvName string) (*resource.Quantity, error) {
 	}
 	return pv.Spec.Capacity.Storage(), nil
 }
+
+// MakePVC Create a PVC and verify that
+//  1. The PVC status transitions to bound,
+//  2. The associated PV is created and its status transitions bound
+//  3. The associated maaystor volume is created and has a State "healthy" if engine is mayastor
+func MakePVC(volSizeMb int, volName string, scName string, volType common.VolumeType, nameSpace string, local bool) (string, error) {
```

Comment:
We should refactor so that MkPVC  calls this function.
MkPVC should be deprecated?

Reply:
I have added deprecated comment here and will be deprecated later https://github.com/openebs/openebs-e2e/pull/23/commits/be7b3f2ee85659903e18b04cf9423c0b9e7cb204